mmcls/datasets/voc.py

```diff
@@ -53,16 +88,19 @@ def load_annotations(self):
                 else:
                     labels.append(label)
 
-            gt_label = np.zeros(len(self.CLASSES))
             # The order cannot be swapped for the case where multiple objects
             # of the same kind exist and some are difficult.
-            gt_label[labels_difficult] = -1
-            gt_label[labels] = 1
+            all_gt_labels[index, labels_difficult] = -1
+            all_gt_labels[index, labels] = 1
 
-            info = dict(
+            sample = dict(
                 img_prefix=self.data_prefix,
                 img_info=dict(filename=filename),
-                gt_label=gt_label.astype(np.int8))
-            data_infos.append(info)
+                gt_label_index=index)
+            samples.append(sample)
 
+        data_infos = dict(
```

Comment:
`data_infos` should be `List[Dict]` instead of `Dict`, it's a protocol, please don't modify it.

Reply:
I understand. Can I instead make `all_gt_labels` a field of `VOC` instance then? I need a place to store common things that can be accessed by all samples in `prepare_data` method.